testsuite/integration/smoke/src/test/config/arq/arquillian.xml

```diff
@@ -7,7 +7,7 @@
     <container qualifier="jboss" default="true">
         <configuration>
             <property name="jbossHome">${jboss.install.dir}</property>
-            <property name="javaVmArguments">${server.jvm.args} -Djboss.inst=${basedir}/target/wildfly</property>
+	    <property name="javaVmArguments">${server.jvm.args} -Djboss.inst=${basedir}/target/wildfly</property>
```

Comment:
Minor: looks like this change should be removed.

Reply:
fixed